core/vm/evm.go

```diff
@@ -187,12 +201,14 @@ func (evm *EVM) Interpreter() Interpreter {
 // the necessary steps to create accounts and reverses the state in case of an
 // execution error or failed value transfer.
 func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas uint64, value *big.Int) (ret []byte, leftOverGas uint64, err error) {
+	log.Debug("~~~ New Call ~~~", "Contract caller:", hex.EncodeToString(caller.Address().Bytes()), "Contract target address:", hex.EncodeToString(addr.Bytes()), "\nCalldata:", hex.EncodeToString(input))
```

Comment:
👌 

Note: the `\n` will make the calldata logged in two different print lines in AWS, meaning searching for `New Call` will filter out the calldata we may be interested in. That said, I think a more global logging solution to this problem may be in order (like we did in typescript, replacing `\n` with `<\\n>`